src/frontend/screens/Settings/components/AnalyticsDialog.tsx

```diff
@@ -0,0 +1,82 @@
+import ContextProvider from 'frontend/state/ContextProvider'
+import React, { useContext } from 'react'
+import { useTranslation } from 'react-i18next'
+
+const STORAGE_KEY = 'analytics-modal-shown'
+
+export default function AnalyticsDialog() {
+  const { showDialogModal } = useContext(ContextProvider)
+
+  const { t } = useTranslation()
+
+  React.useEffect(() => {
+    if (!localStorage.getItem(STORAGE_KEY)) {
+      showDialogModal({
+        showDialog: true,
+        title: t('analyticsModal.title', 'Send Anonymous Analytics'),
+        message: (
+          <>
+            {t(
+              'analyticsModal.info.pt1',
+              'In order to improve the app, Heroic collects 100% anonymous data.'
+            )}
+            <ul>
+              <li>
+                {t(
+                  'analyticsModal.info.pt2',
+                  'Heroic uses the open-source Plausible Analytics platform to gather basic data like: App Version, OS, Stores and Country.'
```

Comment:
```suggestion
                  'Heroic uses the open-source Plausible Analytics platform to gather basic data: App Version, OS, Stores and Country.'
```

to make it clear it's only that and nothing more maybe? because `like ...` sounds like there are other things that we are not listing